frontend/javascripts/admin/voxelytics/workflow_list_view.tsx

```diff
@@ -187,26 +200,26 @@ export default function WorkflowListView() {
           },
           {
             title: "User",
-            dataIndex: "username",
-            key: "user",
-            filters: uniqueify(renderRuns.map((run) => run.username)).map((username) => ({
-              text: username,
-              value: username,
+            key: "userName",
+            dataIndex: "userDisplayName",
+            filters: uniqueify(renderRuns.map((run) => run.userDisplayName)).map((username) => ({
+              text: username || "",
+              value: username || "",
             })),
             onFilter: (value: string | number | boolean, run: RenderRunInfo) =>
-              run.username.startsWith(String(value)),
+              run.userDisplayName?.startsWith(String(value)) || false,
             filterSearch: true,
           },
           {
-            title: "Hostname",
+            title: "Host",
             dataIndex: "hostname",
```

Comment:
The prop `hostname` was renamed to `hostName` right? But not here? is this maybe something?